app/controllers/courses_controller.rb

```diff
@@ -11,6 +11,7 @@ def index; end
   def show; end
 
   def update
+    @course.set_all_attributes_dirty
```

Comment:
Just to ask, why not just suppress the update completely if there are no changes?

Reply:
@heidar and I just did a spike of a solution in the backend to allow for empty attributes using `fetch`. We think this would be a cleaner approach than this ... I'll cut a PR for it.